csv2wiki/__init__.py

```diff
@@ -805,6 +787,42 @@ def _do_skel(self, skel, page_title, row, row_num):
             + " " + ("=" * skel.level)                             \
             + text
 
+    def _wikiize_cell(self, page_title, data_row, col_idx):
+        """Update a CELL to be ready for the wiki.
```

Comment:
Please document every argument.  It's not just a formalism -- here, it's not obvious what DATA_ROW is and how it's used, or how COL_IDX is used.

Reply:
Ack!  Good catch!  V1 of this method was self, and cell.  Coding speed was greater than documenting speed